thespoon/src/components/restaurantPage/AddMenuModal.js

```diff
@@ -42,7 +65,7 @@ class AddMenuModal extends Component {
     //</editor-fold>
 
     //<editor-fold desc="Bussiness Logic">
-    handleSubmit = (event) => {
+    handleSubmit = event => {
```

Comment:
Why do we remove the brackets here? I usualy would put event into brackets to indicate its purpose as a parameter (in the same way as you put parameters in brackets in Java, Python, Swift or C).